src/main/scala/uk/gov/nationalarchives/referencegenerator/Encoder.scala

```diff
@@ -4,14 +4,15 @@ import uk.gov.nationalarchives.oci.Alphabet.Alphabet
 import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
 
 object Encoder {
-  val alphabet: Alphabet = Alphabet.loadAlphabet(Right(IncludedAlphabet.GCRb25)) match {
-    case Left(error) => throw new Exception(s"Error loading the GCRb25 encoding alphabet", error)
+  val alphabet: Alphabet = Alphabet.loadAlphabet(Right(IncludedAlphabet.CTDb25)) match {
+    case Left(error) => throw new Exception(s"Error loading the CTDb25 encoding alphabet", error)
     case Right(alphabet) => alphabet
   }
   val baseNumber = 25
 
   def encode(counter: Long): String = {
     val alphabetIndices = BaseCoder.encode(counter, baseNumber)
-    Alphabet.toString(alphabet, alphabetIndices)
+    val prefix = 'Z'
```

Comment:
Could you move the `prefix` val out of the method and make it an object val as per the ``baseNumber`.

Also maybe rename it to `referencePrefix`